code/modules/surgery/organs/augments_arms.dm

```diff
@@ -98,23 +98,26 @@
 		var/obj/item/assembly/flash/F = holder
 		F.set_light(7)
 
-	var/obj/item/arm_item = owner.get_active_held_item()
-
-	if(arm_item)
-		if(!owner.dropItemToGround(arm_item))
-			to_chat(owner, "<span class='warning'>Your [arm_item] interferes with [src]!</span>")
+	var/side = zone == BODY_ZONE_R_ARM? "r" : "l"
```

Comment:
While you're at it can you make left/right side defines and kill these pointless static lists in `get_empty_held_index_for_side`

Reply:
done